libcylinder/src/signing/mod.rs

```diff
@@ -212,6 +235,18 @@ impl<'a> Signer<'a> {
     }
 }
 
+impl<'a> super::Signer for Signer<'a> {
```

Comment:
This seems like we should rename the existing signer to remove any confusion here.

Reply:
Renamed to `ContextSigner`